layouts/partials/fragments/content.html

```diff
@@ -9,7 +9,7 @@
     <div class="container py-5">
       <div class="row">
         {{- with .Params.sidebar -}}
-          <div class="col-md-3 pb-4">
+          <div class="col-md-3 pb-4 {{- if (eq .align "right") }} order-1 {{- else }} order-0 {{- end -}}">
```

Comment:
We might be able to use something similar for better alignment code in the other fragments. Mostly the `item` fragment. Could lead to less code duplication.